webpack/ForemanTasks/Components/TasksDashboard/Components/ScheduledTasksCard/ScheduledTasksCard.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Card, Icon } from 'patternfly-react';
+import classNames from 'classnames';
+import { translate as __ } from 'foremanReact/common/I18n';
+import './ScheduledTasksCard.scss';
+
+const ScheduledTasksCard = ({ className, scheduled, focusedOn, onClick }) => (
+  <Card
+    className={classNames(
+      'tasks-donut-card',
+      'scheduled-tasks-card',
+      className,
+      {
+        'selected-tasks-card': focusedOn && focusedOn.total,
+      }
+    )}
+  >
+    <Card.Title onClick={onClick}>{__('Scheduled')}</Card.Title>
+    <Card.Body>
+      <div
+        className={classNames('scheduled-data', {
+          'not-focused': !(focusedOn && (focusedOn.total || focusedOn.normal)),
+        })}
+        onClick={onClick}
+      >
+        {scheduled}
+        <Icon type="fa" name="clock-o" />
+      </div>
+    </Card.Body>
+  </Card>
+);
+
+ScheduledTasksCard.propTypes = {
+  className: PropTypes.string,
+  focusedOn: PropTypes.shape({ total: PropTypes.bool, normal: PropTypes.bool }),
```

Comment:
It makes more sense to me to use `focusedOn: PropTypes.oneOf(['TOTAL', 'NORMAL', 'NONE']),`

1. It makes it more clear to look at the propTypes and understand the different states of the component.

2. It will be easier to understand the the the `classNames`
```js
className={classNames('scheduled-data', {
  'not-focused': focusedOn === 'NONE',
})}

className={classNames(
  'tasks-donut-card',
  'scheduled-tasks-card',
  className,
  {
    'selected-tasks-card': focusedOn === 'TOTAL',
  }
)}
```

Reply:
but with `consts`

Reply:
The focusedOn object is build to work well with the received API data, and to allow to select multiple  queries (stopped - warning and stopped error for example). It needs to be an object so it will work the same with all the cards.

Reply:
The `focusedOn` is not going to be data received from the API so we got more freedom about the datatypes we are using.